moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -81,13 +81,14 @@ export const useAdvancedSearch = () => {
     const advancedFilters = makeAdvancedSearchWhereFilters(filters);
     if (advancedFilters.length === 0) return null;
 
+    const bracketedFilters = advancedFilters.map((filter) => `{ ${filter} }`);
+
     if (isOr) {
       // Ex. _or: [{project_lead: {_eq: "COA ATD Project Delivery"}}, {project_sponsor: {_eq: "COA ATD Active Transportation & Street Design"}}]
-      const bracketedFilters = advancedFilters.map((filter) => `{ ${filter} }`);
       return `_or: [${bracketedFilters.join(",")}]`;
     } else {
       // Ex. project_lead: {_eq: "COA ATD Project Delivery"}, project_sponsor: {_eq: "COA ATD Active Transportation & Street Design"}
-      return advancedFilters.join(", ");
+      return `_and: [${bracketedFilters.join(",")}]`;
```

Comment:
Noting that this wrapper applies when there is only one filter too. I tested with the aggregate query and saw consistent totals with and without the `_and` operator. Wanted to call it out in case I am overlooking something.

Reply:
I have definitely used the _and: with only one field name while messing around in hasura console's graphiql and it works there too. basically I'm saying I concur with what youve seen

Reply:
it would be interesting to see how this translates to SQL and if there is any performance hit. but it seems to be working fine and i am not going to do that :)